### PR TITLE
Update dependency version for kserve and istio

### PR DESCRIPTION
**Pull request by pradithya**
Signed-off-by: Pradithya Aria <pradithya.pura@go-jek.com>

**What this PR does / why we need it**:

This PR updated dependency for kserve and istio. 
1. Kserve : Changed the `InferenceService` version from v1alpha2 to v1beta1
2. Istio: Changed the `VirtualService` version from v1alpha3 to v1beta1

Note: 
- There is a breaking changes in PyTorch as v1beta1 InferenceService uses [torchserve](https://pytorch.org/serve/)
- The target branch of this PR is `v1beta1` branch. 

**Which issue(s) this PR fixes**:

Fixes #

**Does this PR introduce a user-facing change?**:

```release-note

```

**Checklist**

- [x] Added unit test, integration, and/or e2e tests
- [x] Tested locally

### CODE REVIEW

File: python/sdk/merlin/model.py [hunk 1]
```diff
@@ -873,7 +874,6 @@ def log_pyfunc_model(self, model_instance, conda_env, code_dir=None,
             raise ValueError(
                 "log_pyfunc_model is only for PyFunc and PyFuncV2 model")
 
-        validate_model_dir(self._model.type, ModelType.PYFUNC, None)
```
Comment (tiopramayudi) [inline python/sdk/merlin/model.py:876]: why removing this?

File: api/cluster/templater.go [hunk 2]
```diff
@@ -126,94 +125,120 @@ func createPredictorSpec(modelService *models.Service, config *config.Deployment
...
 	case models.ModelTypeTensorflow:
-		predictorSpec = kfsv1alpha2.PredictorSpec{
-			Tensorflow: &kfsv1alpha2.TensorflowSpec{
-				StorageURI: utils.CreateModelLocation(modelService.ArtifactURI),
-				Resources:  Resources,
+		predictorSpec = kservev1beta1.PredictorSpec{
+			Tensorflow: &kservev1beta1.TFServingSpec{
+				PredictorExtensionSpec: kservev1beta1.PredictorExtensionSpec{
+					StorageURI: &storageUri,
+					Container: corev1.Container{
```
[the remark this answers is not in the document]
Comment (tiopramayudi) [reply api/cluster/templater.go:148]: also can you allow it through SDK

File: python/sdk/merlin/model.py (hunk 1 above)
Comment (pradithya) [reply python/sdk/merlin/model.py:876]: Do we need this? I don't see any validation necessary for pyfunc model

File: api/cluster/templater.go (hunk 2 above)
[the remark this answers is not in the document]
Comment (pradithya) [reply api/cluster/templater.go:148]: I think I'll create separate PR/Backlog for that.

**Pull request merged**